### PR TITLE
fix: introduce disabledInteractive property and revert focusing disabled actions in general

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/elements/expansion-panel/expansion-panel-header/readme.md [hunk 1]
```diff
@@ -31,18 +31,30 @@ The component can be displayed in `disabled` state using the self-named property
 
 When the element is clicked, the `toggleExpanded` event is emitted.
 
+## Accessibility
+
+### Interactive disabled buttons
+
+Native disabled `<button>` elements cannot receive focus and do not dispatch any events. This can
```
Comment (jeripeierSBB) [inline src/elements/expansion-panel/expansion-panel-header/readme.md:38]: Maybe adapt text to components

File: src/elements/link/block-link-button/readme.md [hunk 2]
```diff
@@ -44,28 +44,30 @@ The component has three sizes (`xs`, `s`, which is the default, and `m`).
...
-### Disabled buttons
+### Interactive disabled Buttons
 
-Generally speaking, `disabled` elements are considered a bad pattern for accessibility. They are invisible to assistive
-technology and do not provide the reason for which they are disabled.
-To partially address the problem, disabled elements are kept focusable (other interactions are still prevented).
-However, it is still the consumers responsibility to provide a reason for the element being disabled.
+Native disabled `<button>` elements cannot receive focus and do not dispatch any events. This can
+be problematic in some cases because it can prevent the app from telling the user why the button is
+disabled. Consumers can use the `disabledInteractive` property to style the button as disabled but allow for
```
Comment (jeripeierSBB) [inline src/elements/link/block-link-button/readme.md:51]: needs to be adapted as there is no disabled style

File: src/elements/link/link-button/readme.md [hunk 3]
```diff
@@ -30,27 +30,29 @@ accepting its associated properties (`type`, `name`, `value` and `form`).
...
-### Disabled buttons
+### Interactive disabled Buttons
 
-Generally speaking, `disabled` elements are considered a bad pattern for accessibility. They are invisible to assistive
-technology and do not provide the reason for which they are disabled.
-To partially address the problem, disabled elements are kept focusable (other interactions are still prevented).
-However, it is still the consumers responsibility to provide a reason for the element being disabled.
+Native disabled `<button>` elements cannot receive focus and do not dispatch any events. This can
+be problematic in some cases because it can prevent the app from telling the user why the button is
+disabled. Consumers can use the `disabledInteractive` property to style the button as disabled but allow for
```
Comment (jeripeierSBB) [inline src/elements/link/link-button/readme.md:37]: Needs adaption, no disabled style

File: src/elements/menu/menu/menu.ts [hunk 4]
```diff
@@ -148,7 +148,13 @@ export class SbbMenuElement extends SbbNamedSlotListMixin<
       this.querySelectorAll<SbbMenuButtonElement | SbbMenuLinkElement>(
         'sbb-menu-button, sbb-menu-link',
       ),
-    ).filter((el: SbbMenuButtonElement | SbbMenuLinkElement) => interactivityChecker.isVisible(el));
+    ).filter(
+      (el: SbbMenuButtonElement | SbbMenuLinkElement) =>
+        (!el.disabled ||
+          (el.localName === 'sbb-button-element' &&
+            (el as SbbMenuButtonElement).disabledInteractive)) &&
```
Comment (jeripeierSBB) [inline src/elements/menu/menu/menu.ts:155]: Needs adaption if link receives disabeldInteractive too

**Review by TomMenga — APPROVED** [review]
LGTM 👍

**Pull request merged**